rag/indexer/embedder.py

```
@@ -0,0 +1,293 @@
+if __name__ == "__main__":
+    logging.info("--- Running Embedder Standalone ---")
+
+    # Example: Test the new JSON function
+    logging.info("Testing JSON embedding function...")
+    if not os.path.exists(DEFAULT_JSON_DIR):
+         logging.warning(f"JSON directory '{DEFAULT_JSON_DIR}' not found.")
+         logging.warning("Please ensure JSON files exist (e.g., from crawl4ai).")
+    else:
+        embedded_json_chunks = load_process_embed_json()
+        if embedded_json_chunks:
+            logging.info(f"Example embedded JSON chunk data structure:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)